src/auth/jwtclient.ts

```diff
@@ -168,6 +168,20 @@ export class JWT extends OAuth2Client {
    * @private
    */
   async refreshToken(refreshToken?: string|null): Promise<GetTokenResponse> {
+    this.createGToken();
+    const token = await this.gtoken!.getToken();
```

Comment:
What about making `createGToken()` return a string? Then you won't need the non-null annotation here.

Reply:
👍

Reply:
Not done here and in line 177 below ?

Reply:
Ooops, sorry bout that.  Cleared it up.